src/ContactPeople/ContactPeopleList.js

```diff
@@ -36,9 +36,14 @@ const getContactsUrl = (orgId, contactId) => {
   return `${starting}${ending}`;
 };
 
-const AddContactButton = ({ fields, stripes, orgId }) => {
-  const addContacts = contacts => {
-    fields.push(...contacts.map(contact => contact.id));
+const AddContactButton = ({ fetchContacts, fields, stripes, orgId }) => {
+  const addContacts = (contacts = []) => {
+    const addedContactIds = new Set(fields.getAll());
```

Comment:
I don't see big sense in converting to set here as you checks containing id below

only case if 2 the same contacts where saved before

Reply:
it's done for performant operation 'has' of set below